pkgs/tools/networking/guardian-agent/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildGoPackage
+, fetchFromGitHub
+, lib
+, openssh
+, autossh
+, x11_ssh_askpass
+}:
+
+buildGoPackage rec {
+  pname = "guardian-agent";
+  version = "0.7.2-beta";
+
+  src = fetchFromGitHub {
+    owner = "StanfordSNR";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "05269y944hcabn6dqa66387rdhx81vcqcyjv6m1hdbz5ba8j7mqn";
+  };
+
+  goPackagePath = "github.com/StanfordSNR/guardian-agent";
+
+  goDeps = ./deps.nix;
+
+  nativeBuildInputs = [ openssh autossh x11_ssh_askpass ];
+
+  modSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
+
+  postInstall = ''
+    mkdir -p $bin/bin $out/share/doc/${pname}
+    cp -v ./go/src/github.com/StanfordSNR/guardian-agent/scripts/* $bin/bin/
+    cp -vr ./go/src/github.com/StanfordSNR/guardian-agent/* $out/share/doc/guardian-agent
```

Comment:
This is catching a lot more than we want

<details>
<summary>Files</summary>

```
❯ tree result
result
├── nix-support
│   └── propagated-build-inputs
└── share
    └── doc
        └── guardian-agent
            ├── agent.go
            ├── animation.gif
            ├── AUTHORS
            ├── cmd
            │   ├── sga-guard-bin
            │   │   └── sga-guard-bin.go
            │   ├── sga-ssh
            │   │   └── sga-ssh.go
            │   └── sga-stub
            │       └── sga-stub.go
            ├── common.go
            ├── delegated_client.go
            ├── doc
            │   ├── AfterHandoff.png
            │   ├── AfterHandoff.xml
            │   ├── agent-stack2.png
            │   ├── badflow2.png
            │   ├── bad flow.vsdm
            │   ├── Connectivity.png
            │   ├── design.md
            │   ├── good flow.vsdm
            │   ├── good flow with lines.vsdm
            │   ├── SSHLevel.png
            │   ├── SSHLevel.xml
            │   └── Untitled Diagram.xml
            ├── LICENSE
            ├── policy.go
            ├── README.md
            ├── release
            │   └── makefile
            ├── scope.go
            ├── scripts
            │   ├── sga-env.sh
            │   └── sga-guard
            ├── socket.go
            ├── socket_windows.go
            ├── sshfwd.go
            ├── store.go
            ├── ui.go
            └── vendor
                └── golang.org
                    └── x
                        └── crypto

15 directories, 33 files
```
</details>

I think that the following files/directories would be appropriate there
* AUTHORS
* doc/
* LICENSE
* README.md